1-the-basics/1-the-basics.html

```diff
@@ -0,0 +1,84 @@
+<!OCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;300;400;700&display=swap" rel="stylesheet">
+    <title>Html Form</title>
+    <link rel="stylesheet" href="./style.css">
+      <body>
+    <header class="header"><h1>Simple Form</h1></header>
+    <main>
+      <img src="https://pixabay.com/get/g1eb23bdbe92babbf01a9cf0b18d90f48e925cd86bc7db09beb7b364e2a88c603870a48172823c4eae7b06567590edc4f0c3b4cb4a47a2fa569a125f1079a460de2a88a67b84cc3e926048cd698b6e33a_640.jpg" alt="prueba">
+      <form class="form">
+        <label for="name"
+          >First Name <br />
+          <input type="text" name="first-name" id="fn" required />
+        </label>
+        <label for="lastname">
+          Last Name <br />
+          <input type="text" name="lastname" required
+        /></label>
+        <label for="email">
+          Email <br />
+          <input type="email" name="email" required
+        /></label>
+        <label for="dateOfBirth">
+          Date of Birth <br />
+          <input type="date" name="date" required
```

Comment:
```suggestion
        <label for="dateOfBirth">
          Date of Birth <br />
          <input type="date" name="dateOfBirth" required
```

Reply:
How should you be? dateOfBirth or date-of-birth?